src/fides/core/config/security_settings.py

```diff
@@ -16,42 +15,93 @@
 ENV_PREFIX = "FIDES__SECURITY__"
 
 
-class SecurityEnv(Enum):
-    """
-    Defines the potential environments that Fides can
-    be configured to run in.
-    """
-
-    DEV = "dev"
-    PROD = "prod"
```

Comment:
Would be curious to hear more about `Enum` vs validator change here for future as well

Reply:
This was specifically because the enum looks really ugly when printed, it comes out as `<SecurityEnum: PROD "prod">` which is not what we wanted...I guess I could override the `__str__` for that class but that seems like a bit much